sdk/core/core-lro/test/utils/router/paramRoutes.ts

```diff
@@ -224,7 +224,7 @@ function deleteasyncRetry(request: PipelineRequest): PipelineResponse | undefine
   return undefined;
 }
 
-// eslint-disable-next-line @azure/azure-sdk/ts-no-namespaces
+// eslint-disable-next-line @typescript-eslint/no-namespace
 namespace deleteasyncRetry {
   export let internalCounter: number = 1; // eslint-disable-line prefer-const
 }
```

Comment:
This is neither here nor there for this PR but the namespaces in these files are the only namespace declarations that actually stick out to me. CC @sadasant and @deyaaeldeen should we file an issue to rework these?

Reply:
I was trying to mimic static variables based on this recommendation: https://github.com/microsoft/TypeScript/issues/8419#issuecomment-216351820 and did not want to change the whole setup of routes logic as functions to accommodate stateful ones.

I am fine with opening an issue though to track updating this.

Reply:
Ah I see the purpose of the pattern now is to let the namespace merge with the function and bind those static fields to it that way. It would be nice to find a different way to bind these to instances of the function (using `Object.assign` and explicitly giving the function's name an extended type might work, or even a class that `extends Function`), but it doesn't matter too much.